### PR TITLE
DEPT-265 ~ Config split for development domain hostnames

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

**Review by johangant — CHANGES_REQUESTED** [review]
LGTM, just needs a tweak for the protocol/scheme for some domain records.

File: config/development/domain.record.group_9.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+uuid: 01ed564b-0795-4cee-93ca-f1cce330813e
+langcode: en
+status: true
+dependencies: {  }
+id: group_9
+domain_id: 16605160
+hostname: infrastructure-ni.dept-edge-3e7cfpi-dnvkwx4xjhiza.uk-1.platformsh.site
+name: Infrastructure
+scheme: variable
```
[the remark this answers is not in the document]
Comment (omahm) [reply config/development/domain.record.group_9.yml:9]: The hostname doesn't have a protocol, see https://github.com/dof-dss/nicsdru_dept/blob/development/config/sync/domain.record.group_1.yml

[the remark this answers is not in the document]
Comment (johangant) [reply config/development/domain.record.group_9.yml:9]: Sorry, meant the `scheme` which is sometimes set to `variable`. Changing `variable` to the value of `https` (or vice versa) across all domain records would keep it consistent.

[the remark this answers is not in the document]
Comment (omahm) [reply config/development/domain.record.group_9.yml:9]: That's a good point, hadn't spotted that.

**Pull request merged**